src/lumen/web/server.hpp

```diff
@@ -9,7 +11,7 @@ struct Server {
     httpd_handle_t server = nullptr;
     httpd_config_t config = HTTPD_DEFAULT_CONFIG();
 
```

Comment:
Last picky question: should the methods be defined before the data members?

Reply:
Not in structs.